### DISCUSSION TITLE
Community video for MLH INIT Hackathon

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#1883] [Ideas] [answered]
Would you like to be in our community video?

**EXAMPLES**
- Previous video https://twitter.com/eddiejaoude/status/1378668010676838403
- **This video style is what we are doing this time** as suggested by @starlightknown https://www.youtube.com/watch?v=QI6qSrBs6BA

**VIDEO CRITERIA**
- write 1-3 words in large text, that answers the question `why should you join EddieHub`
- A4 paper, folded in half
- recored in landscape

**FLOW OF YOUR VIDEO**
1. receive the paper from YOUR left
2. open paper and show text on paper
3. fold paper and pass to YOUR right
4. upload to google drive or dropbox and share a link below in this thread https://github.com/EddieHubCommunity/support/discussions/1883#discussioncomment-906770

**TIPS**
- have more light behind the camera facing you
- use the highest resolution setting possible
- high frame rate, like slow mo is NOT needed

**TEXT EXAMPLES**
- EddieHub FTW
- Collaboration 1st Code 2nd
- Community FTW
*But please use your own words*

**DEADLINE: Wednesday 23rd of June 6pm BST**

**Comment by iason9** [comment]
Yes, count me as One!

**Reply by eddiejaoude** [reply]
Great 👍

**Comment by kaiwalyakoparkar** [comment]
Count me too. I guess you forgot to add the deadline tho.

**Reply by kaiwalyakoparkar** [reply]
I am going to write "It not just community, It's family"

**Reply by eddiejaoude** [reply]
Awesome 👍

**Comment by verma-kunal** [comment]
I am in too 😃

**Reply by kaiwalyakoparkar** [reply]
Please do let everyone know what you will be writing on the paper so that everyone can avoid repeatition :wink:

**Reply by verma-kunal** [reply]
i would be writing "Learning & Empowering Others" !! With some designs/logo here n there 🙂

**Reply by eddiejaoude** [reply]
Cool 👍

**Comment by archismansaha** [comment]
I am in . where we will get to know what others are writing and submit our video .
I am planning to write  "Everytime to help!" will it be relevant?

**Reply by kaiwalyakoparkar** [reply]
You will probably get to see the videos by other and there's quotes here itself. BTW great choice of quote :)

**Reply by archismansaha** [reply]
Ohk another question where have to send the videos ?? In discord or here?

**Reply by kaiwalyakoparkar** [reply]
Here, would be more good

**Comment by muthuannamalai12** [comment]
I am in 

I am planning to write Alone we can do so little; together we can do so much.

Is it okay??

**Reply by kaiwalyakoparkar** [reply]
Yup that sounds amazing

**Reply by muthuannamalai12** [reply]
Thank you

**Reply by eddiejaoude** [reply]
Nice 👍

**Comment by eddiejaoude** [comment, marked as answer]
Video links only please 👇

**Reply by iason9** [reply]
Hope its good : https://drive.google.com/file/d/1FUT3cG_INcYrhlHUdNHnXsFsvId0dwUh/view?usp=sharing

**Reply by archismansaha** [reply]
Is it ok : https://drive.google.com/file/d/1Yp0x1d5z52I43Bg21nZVtQr0HuRFooDV/view?usp=drivesdk

**Reply by kaiwalyakoparkar** [reply]
Hope that's good: https://drive.google.com/file/d/15xgu5iuNcPF4Nin0x_AEAvFOenpZMVeW/view?usp=sharing 
I tried my best to make it. Sorry if something went wrong :sweat_smile:

**Reply by starlightknown** [reply]
https://drive.google.com/file/d/1Sbo49G28_jAhbrALPcIeEAz6wS2WYVwX/view?usp=drivesdk
Ignore my drawings xD

**Reply by ashwinexe** [reply]
https://drive.google.com/file/d/182uQHBN2l3KTfi-ZJi4a7cqw7INq24c5/view?usp=sharing
My drawing's still rusty, its been years since I doodled 😃

**Reply by verma-kunal** [reply]
Link: https://drive.google.com/file/d/1tuKbk7G_vdXjZ_oUg0rm_jlodDG1LhtS/view?usp=sharing
Lighting couldn't be that good !! Hope the rest is fine 🤞

**Reply by WebShivam** [reply]
Link : https://drive.google.com/file/d/1xIIMpxm314L8fKzQLLTirNWNZxvQD5Gy/view?usp=drivesdk
My handwriting is bad,  so I spent whole 1hour to make it atleast readable 😆
Hope video is okay?
Your suggestions are welcome :)

**Reply by alex-anson** [reply]
https://www.dropbox.com/s/12lzvpfj3nr9mme/IMG_8003.MOV?dl=0

Lemme know if there's any issue! 😁
I did it 💪🥳

**Reply by eddiejaoude** [reply]
All are great 💥 thank you ❤️ 

1. [x] @starlightknown yours is passing from the wrong direction 
2. [x] @ashwinexe yours is passing from the wrong direction
3. [x] @WebShivam yours is private so I cannot see / download it

**Reply by starlightknown** [reply]
Can't the video flipped in editting? I thought it should go from left to right 🤦🏻‍♀️

**Reply by eddiejaoude** [reply]
yes the video can be flipped, but the text will be backwards

**Reply by starlightknown** [reply]
https://drive.google.com/file/d/1T3pNgmfkXbBIhg3hye_lMl40RQwTosSY/view?usp=drivesdk 
Hope it's ok now

**Reply by Dhriti-Roy** [reply]
https://drive.google.com/file/d/1-kEFs3nwRF3ke8CO3U1JI7XuGslM9ut2/view?usp=drivesdk
Hope it's fine. Let me know if there's anything issue. 😇

**Reply by eddiejaoude** [reply]
Thank you both for updating.

- @starlightknown looks good now 👍 
- @Dhriti-Roy looks good but is still in portrait not landscape mode - we can try to use it, but I am not sure until we starting editing

**Reply by ashwinexe** [reply]
> All are great 💥 thank you ❤️
> 
> 1. [ ]  @starlightknown yours is passing from the wrong direction
> 2. [ ]  @ashwinexe yours is passing from the wrong direction
> 3. [x]  @WebShivam yours is private so I cannot see / download it

Thanks for the heads up Eddie! 😲 I've updated the video over the same link 😁

**Reply by muthuannamalai12** [reply]
My video Link: https://drive.google.com/file/d/14Od26I1VYN8qt4f8Boqc-jaNkMCmBiGT/view?usp=sharing

**Reply by Dhriti-Roy** [reply]
https://drive.google.com/file/d/1xZArJ9tVYccJE09FJ7ZBl8oouWqLiC-G/view?usp=sharing
I hope this works. I have edited it from portait to landscape.

**Reply by eddiejaoude** [reply]
thank you @muthuannamalai12 , some feedback

- someone's finger is on the bottom left of the camera
- the paper is not in the video frame when you receive it or pass it (please watch the example at the top of the discussion)

thank you @Dhriti-Roy for re-recording 👍

**Reply by eddiejaoude** [reply]
Thank you all 🔥 . I am going to lock this discussion now, so we can get video editing. We will do the best with all the videos and try to get them all in some how even if recorded incorrectly (but we cannot promise)

**Comment by alex-anson** [comment]
I think I'm gonna skip this one... I'm not quite up to the task yet!
Someone can take mine -- 

Inclusivity and inspiration.

:)

**Reply by eddiejaoude** [reply]
Sure, no problem, when you are ready we are here 🤓

**Reply by alex-anson** [reply]
Thank you Eddie!

**Reply by alex-anson** [reply]
I was letting my anxiety get the best of me. I'm in!

**Reply by kaiwalyakoparkar** [reply]
That's awesome. I really appreciate how you are beating over your fears and stepping out of comfort zone.

**Reply by verma-kunal** [reply]
> I was letting my anxiety get the best of me. I'm in!

Thats really great to hear !! We are all together in this

**Reply by ashwinexe** [reply]
> I was letting my anxiety get the best of me. I'm in!

Awesome!

**Reply by eddiejaoude** [reply]
OMG amazing if you are in 💥

**Reply by alex-anson** [reply]
I'm totally in. 😁🎉

**Reply by WebShivam** [reply]
> I'm totally in. 😁🎉

Awesome 🎉🙌

**Reply by kaiwalyakoparkar** [reply]
You did awesome :boom:

**Reply by verma-kunal** [reply]
yay ! you made it @alex-anson !! Thats awesome 🎉

**Reply by alex-anson** [reply]
Thanks so much for the encouragement, everybody! It means a lot. ☺️

**Comment by Dhriti-Roy** [comment]
Hey @eddiejaoude , can I also be a part of this ?

**Reply by verma-kunal** [reply]
Hey @Dhriti-Roy !! Surely you can be a part of this 🙂
Do read the rules for making the clip as mentioned above & send in the drive link **(pls note the deadline as well )**

**Reply by Dhriti-Roy** [reply]
When is the deadline ?

**Reply by ashwinexe** [reply]
> When is the deadline ?
It says right under the instructions (tonight 10:30 pm IST) :)

**Reply by eddiejaoude** [reply]
Yes @Dhriti-Roy join us 👍 

You have a few hours still 🎉

**Comment by WebShivam** [comment]
How about writing "Fun, Networking and Skills" ?

**Reply by ashwinexe** [reply]
That's a good one

**Reply by verma-kunal** [reply]
> Link : https://drive.google.com/file/d/1xIIMpxm314L8fKzQLLTirNWNZxvQD5Gy/view?usp=drivesdk
> My handwriting is bad, so I spent whole 1hour to make it atleast readable 😆
> Hope video is okay?
> Your suggestions are welcome :)

hey @WebShivam , you would have to make the drive link as "un-restricted" so that anyone with the link can have access to that !!
Right now, its restrcted

**Reply by WebShivam** [reply]
Is it unrestricted now @verma-kunal ? Lemme know

**Reply by ashwinexe** [reply]
> Is it unrestricted now @verma-kunal ? Lemme know  

No, it still asks for permission. You can check `anyone with link can : view` is turned on

**Reply by WebShivam** [reply]
It should work now I did the changes few minutes ago. Can you check it please?

**Reply by kaiwalyakoparkar** [reply]
@WebShivam Don't worry. I guess it's working now

**Reply by eddiejaoude** [reply]
Yep it is working now 👍

**Reply by WebShivam** [reply]
Thank you @kaiwalyakoparkar @eddiejaoude ☺️

**Comment by ashwinexe** [comment]
I am thinking to quote "Inclusive, welcoming and safe"

**Reply by WebShivam** [reply]
Yeah Noice Go For It 👍

**Comment by starlightknown** [comment]
I'm thinking about "join us in MLH INIT and <date>" is this taken?

**Reply by Dhriti-Roy** [reply]
Well, that's not taken, I guess.

**Reply by Dhriti-Roy** [reply]
I would love to complete the sentence by writing " let's begin the Celebration". 
What you think ?

**Reply by kaiwalyakoparkar** [reply]
@Dhriti-Roy Go for it. The last few hours remaining. You can do it!

**Reply by eddiejaoude** [reply]
thank you @Dhriti-Roy, some feedback

- please can you record in landscape (not portrait)
- please have the paper folded and passed to you from the left to the right (not thrown)
- add the link to the link thread [here](https://github.com/EddieHubCommunity/support/discussions/1883#discussioncomment-906770) (also there are some examples if you need)
